.github/workflows/kubernetes-agent-publish-chart.yaml

```diff
@@ -9,8 +9,6 @@ on:
     - .github/workflows/kubernetes-agent-publish-chart.yaml
 
   pull_request:
-    branches:
-    - '*'
```

Comment:
Allows the publish to run for PR's

Reply:
Yeah this is a security issue isn't it?

Reply:
hrm, good question... I'll ask in #secops-requests...

Reply:
I can revert this while we wait for an answer

Reply:
reverted in 
[f592fe9](https://github.com/OctopusDeploy/helm-charts/pull/81/commits/f592fe92f8be5d9e45d0f2ec5d90f3f9e1b0d8e0)

Reply:
Wait I think I'm confused, so removing the `*` allows it to run on branches? I thought `*` would apply to all branches? 🤔

Anyways, yeah I'm unsure if it's ok - if someone external has access to running github workflows, is that ok?

Reply:
correct, removing this means it runs on all branches. There is some weird thing going on with the `*` not behaving